openmmtools/tests/test_integrators.py

```diff
@@ -247,7 +253,76 @@ def test_external_protocol_work_accumulation():
     n_globals = integrator.getNumGlobalVariables()
     names_of_globals = [integrator.getGlobalVariableName(i) for i in range(n_globals)]
     assert('protocol_work' not in names_of_globals), "Protocol work should not be defined."
-
+    del context, integrator
+
+class TestExternalPerturbationLangevinIntegrator(TestCase):
+    def test_protocol_work_accumulation_harmonic_oscillator(self):
+        """Testing protocol work accumulation for ExternalPerturbationLangevinIntegrator with HarmonicOscillator
+        """
+        testsystem = testsystems.HarmonicOscillator()
+        parameter_name = 'testsystems_HarmonicOscillator_x0'
+        parameter_initial = 0.0 * unit.angstroms
+        parameter_final = 10.0 * unit.angstroms
+        for platform_name in ['Reference', 'CPU']:
+            self.compare_external_protocol_work_accumulation(testsystem, parameter_name, parameter_initial, parameter_final, platform_name=platform_name)
+
+    def test_protocol_work_accumulation_waterbox(self):
```

Comment:
Sure, we can relax the precision, but are you sure you are setting `OPENMM_CPU_THREADS=1` in your shell BEFORE you run the tests? The [travis tests do this](https://github.com/choderalab/openmmtools/pull/168/files#diff-354f30a63fb0907d4ad57269548329e3R41).